tests/app/service/test_rest.py

```diff
@@ -3372,3 +3372,12 @@ def test_get_monthly_notification_data_by_service(mocker, admin_request):
 
     dao_mock.assert_called_once_with(start_date, end_date)
     assert response == []
+
+
+def test_get_returned_letter_summary(admin_request, sample_service):
```

Comment:
Might be worth testing the JSON that is actually returned from the endpoint as well as how many items are in it.